shuup/core/__init__.py

```diff
@@ -49,6 +50,10 @@ def ready(self):
 
             install_error_handlers()
 
+        from .models import SupplierModule
+
+        SupplierModule.ensure_all_supplier_modules()
```

Comment:
what is this for?

Reply:
To make sure that all the SupplierModule objects are created. If they are not the admin would not be able to select them for the supplier